build.gradle.kts

```diff
@@ -110,8 +110,11 @@ tasks["detekt"].dependsOn(tasks["detektE2eTest"])
 
 tasks.register<Task>("prepareReleaseOfUpdatedSchemas") {
     group = "releaseManagement"
+    val skipUnstableVersions = findBooleanProperty("skipUnstableVersions", defaultValue = false)
+    val fastForwardToMostRecentVersion = findBooleanProperty("fastForwardToMostRecentVersion", defaultValue = false)
+
```

Comment:
These properties have good names, so they are rather self explanatory, but having some short comments (with a practical example) would be rellay nice to have.

Code comment here and/or markdown in `docs/development-guidelines.md` 

Some ideas:

- `skipUnstableVersions` - setting this to `true` will skip versions like `4.10.0-alpha.1674847484+ffd6999e`
- `fastForwardToMostRecentVersion` - if multiple new versions have appeared since our last release (e.g. `4.13.0`, `4.14.0`, `4.15.0`), setting this to `true` will force `prepareReleaseOfUpdatedSchemas` to use the most recent one (`4.15.0`)